public/js/action-list.js

```diff
@@ -163,6 +191,22 @@
                 ).addClass('active');
             }
         }
+
+        var $activeItems = $list.find('.list-item.active');
+
+        if ($activeItems.length) {
+            var container = $list.closest('.container');
```

Comment:
Here in `onRendered`, `$target` is already the container. You don't have to query that again.